unittests/InstructionCountCI/FEXOpt/MultiInst_AFP.json

```diff
@@ -24,8 +24,10 @@
         "addss xmm0, xmm2"
       ],
       "ExpectedArm64ASM": [
-        "fadd s16, s16, s17",
-        "fadd s16, s16, s18"
+        "mov v2.16b, v16.16b",
```

Comment:
This one looks pretty bad. Looks like it will completely break scalar SSE operations. It is zeroing the upper bits insert of doing an insert. CI isn't hitting it since nothing in CI supports AFP yet.

Reply:
Yes, that's unfortunately fallout from the tied register issue. I could try to frontload some RA work now to deal with it but it's very much nontrivial. Alternatively, I could maybe see how much of this PR can land without regression and then deal with the rebasing for the next few months... It's not critical things land asap, just trying to avoid ending up with a gigantic "rewrite the world" patch set if I can help it!

Reply:
I've split out #3705 and #3704 as changes that can land now, then what's left here shouldn't be too bad to rebase for a while until I have RA in better shape. Will close this for now while I shave RA shaped yaks / IR reworks.